### PR TITLE
Adding cntk 209 super-resolution tutorial

### PR DESCRIPTION
**Pull request by t-bovuk**
Adding all three notebooks for super-resolution tutorial. There are also two python scripts which can be used to set up the training data. Everything was done by following the logic of the CNTK103 tutorial. There is a test for each notebook.
Notebook A - evaluation of pretrained models
B - data prep
C - model descriptions and training

### CODE REVIEW

File: Tutorials/CNTK_209A_Evaluation_of_Pretrained_Super-resolution_Models.ipynb [hunk 1]
```diff
@@ -0,0 +1,634 @@
...
+     "output_type": "stream",
+     "text": [
+      "Now creating: Data/BerkeleySegmentationDataset/example_results/VDSR_results/example_1.jpg\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/anaconda/envs/py35/lib/python3.5/site-packages/cntk/core.py:74: RuntimeWarning: data is not C contiguous; rearrange your data/computation to avoid costly data conversions\n",
```
Comment (t-bovuk) [inline Tutorials/CNTK_209A_Evaluation_of_Pretrained_Super-resolution_Models.ipynb]: This runtime warning appears, not sure how to avoid it. It does not affect the performance or running of the notebook.

File: Tutorials/CNTK_209A_Evaluation_of_Pretrained_Super-resolution_Models.ipynb [hunk 2]
```diff
@@ -0,0 +1,634 @@
...
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "#load models\n",
+    "VDSR_model = load_model(os.path.join(\"pretrained_models\", \"VDSR.model\"))\n",
```
Comment (t-bovuk) [inline Tutorials/CNTK_209A_Evaluation_of_Pretrained_Super-resolution_Models.ipynb]: We are currently loading models from our VM (this will obviously not work for checking-in). How should loading of the models look like for the check-in? Where do we put the models so they can be loaded in an appropriate way?

File: Tutorials/CNTK_209A_Evaluation_of_Pretrained_Super-resolution_Models.ipynb [hunk 3]
```diff
@@ -0,0 +1,634 @@
...
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from IPython.display import Image\n",
+    "Image(url = \"https://superresolution.blob.core.windows.net/superresolutionresources/example.PNG\")"
```
Comment (t-bovuk) [inline Tutorials/CNTK_209A_Evaluation_of_Pretrained_Super-resolution_Models.ipynb]: We are currently showing images from our blob storage. How do we get images to cntk storage?

File: Tutorials/CNTK_209C_Image_Super-resolution_Using_CNNs_and_GANs.ipynb [hunk 4]
```diff
@@ -0,0 +1,1562 @@
...
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Select the right target device when this notebook is being tested:\n",
+    "if 'TEST_DEVICE' in os.environ:\n",
```
Comment (t-bovuk) [inline Tutorials/CNTK_209C_Image_Super-resolution_Using_CNNs_and_GANs.ipynb]: This notebook should be tested and used only on GPU (look at test file). This is because batch normalization training is not yet implemented on CPU. How should this part look, then? I will also have to add a comment explaining this in the notebook.

File: Tutorials/CNTK_209C_Image_Super-resolution_Using_CNNs_and_GANs.ipynb [hunk 5]
```diff
@@ -0,0 +1,1562 @@
...
+       "('VGG19_ImageNet_Caffe.model', <http.client.HTTPMessage at 0x7fba3c455860>)"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "urlretrieve(\"https://www.cntk.ai/Models/Caffe_Converted/VGG19_ImageNet_Caffe.model\", \"VGG19_ImageNet_Caffe.model\")"
```
Comment (t-bovuk) [inline Tutorials/CNTK_209C_Image_Super-resolution_Using_CNNs_and_GANs.ipynb]: We are taking VGG19 model from cntk.ai location and saving it in the same path as the notebook (not deleting it after use). Notify us if this part should look different.

**Comment by t-bovuk** [comment]
Reorganized notebooks and merged B and C into one, as per Sayan's comments. Tests were reshaped accordingly.

**Comment by KeDengMS** [comment]
Seems super resolution tutorial has been merged.
